src/SSVOpenHexagon/Core/MenuGame.cpp

```diff
@@ -1834,6 +1820,9 @@ void MenuGame::update(ssvu::FT mFT)
         case States::LoadingScreen: hexagonRotation += mFT / 100.f; return;
 
         case States::LevelSelection:
+            focusHeld = sf::Keyboard::isKeyPressed(KKey::LShift) ||
+                        sf::Keyboard::isKeyPressed(KKey::RShift);
```

Comment:
I'm OK with this as a first version, but focus can be rebound, though. We should probably get the binding from the config.

Reply:
Makes sense, I will take care of it next week.

Reply:
Would it be ok if I used the Focus bind for this?
```
add2StateInput(game, Config::getTriggerFocus(), focusHeld, Tid::Focus);
```